cli/cli.ts

```diff
@@ -2300,14 +2302,14 @@ class Host
         check(p)
 
         if (U.endsWith(filename, ".uf2"))
-            fs.writeFileSync(p, contents, "base64")
+            fs.writeFileSync(p, contents, { encoding: "base64" })
```

Comment:
Node.js doesn't seem to deprecate string as encoding usage. Is the @types missing it?

Reply:
Yeah, [you're right](https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback). Not sure why it was removed from the `@types`